extension/extension/background.js

```diff
@@ -189,8 +190,13 @@ function websocket_logger(server) {
     }
 
     return function(data) {
-        queue.push(data);
-        dequeue();
+        switch (authStatus) {
+            case "deny": // don't update/empty the queue if authStatus is "deny"
```

Comment:
This should probably be a keyword instead similar to how you did this on the Python side `DENY = 'deny'`. Then instead checking for `case DENY:`.

Reply:
I have replaced every string constant with their corresponding keywords